circleguard/gui.pyw

```diff
@@ -6,17 +6,17 @@ from queue import Queue, Empty
 from functools import partial
 
 # pylint: disable=no-name-in-module
-from PyQt5.QtCore import Qt, QTimer, QSettings
+from PyQt5.QtCore import Qt, QTimer
```

Comment:
good changes, I'm pretty much always a fan of explicit imports over wildcard imports, just during development it can be a pain to constantly be changing the import statements when you add a function or rename something